sagemathinc/cocalc
# Add signup question

# Description
Add a simple one line question to sign up to help us understand what a user wants when they sign up.

# Testing Steps
1. Create a new account with some data in the field
1. Inside the hosting project, use `psql smc` and the query `select account_id, tracking_info from accounts;`
1. See the data there.
1. Create an account without the field filled out
1. It should still work.

<img width="560" alt="screen shot 2018-12-03 at 6 54 38 pm" src="https://user-images.githubusercontent.com/618575/49388585-e789f580-f72c-11e8-9943-b7e3617cfe52.png">
# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
